### ISSUE TITLE
Cannot Save CityInfo Page

### ISSUE THREAD

**Issue by Jcrowley** [city72/city-72#143]
at url cm/city:

After filling out all required fields, the message "Please double-check that all required fields are filled." remains in place and the Save Button is disabled. Noted with all fields filled out as well. Safari and FireFox on OSX 10.10.

**Comment by arena** [comment]
That seems odd. Was there anything at all missing? Maybe an image?

**Comment by Jcrowley** [comment]
No, I filled in every field, including affiliates. Deployed from GitHub to Heroku, set up admin, went to City Info page, tried repeatedly to get it to take. Deleted Heroku instance, reinstalled. Still same behavior.

**Comment by arena** [comment]
I'm going to try to reproduce with a new deploy and see if I can reproduce the bug. Thanks for reporting it!

**Comment by arena** [comment]
Ok. So, I created a new deploy and I was able to save the page and then view it. I think I know what's going on though. There was a bug that cropped up toward the end of the development cycle related to large images. I was never able to reproduce it myself but the development team knew about it. When you upload a large image and/or are on a slow connection. The upload times out, fails, without any communication of what went wrong. Ugh. I suspect that is the bug you encountered. I will link it to this thread.

Would you mind trying to save the City Info form again but with smaller images to see if that's what's going on?

**Comment by Jcrowley** [comment]
Arena,

Thanks, that was part of the problem. The other bit was the acronym for the org with which I am working is 5 characters long (not 4). I missed that.

**Comment by arena** [comment]
I'm glad you are able to move forward.

A couple of notes:

1. Thanks for reporting this buggy experience. If there were ongoing development, I think this would be the top priority to fix. When I get some time I'll look into these interim solutions:
-  putting some help text next to the images noting the potential for issues with large images and/or slow connections
- putting a note on the abbreviation field about the character limit

2. In your case, if you really must have 5 characters and they'll fit, you could:
- fork the code
- change the character limit
- redeploy from your fork
..but maybe you found a shorter version of the name that you are happy with. 

Good luck with your project and please post any other bugs you find. A few people watch this project and I know there are some developers that would like to contribute when they have the time, so the bug activity is good (even though the bugs are annoying).

I'd love to see your project when you are ready to share it.
